go.mod

```diff
@@ -0,0 +1,53 @@
+module code-review-bot
+
+go 1.23.4
+
+require (
+	github.com/bep/clocks v0.5.0 // indirect
+	github.com/bep/godartsass/v2 v2.3.2 // indirect
+	github.com/bep/golibsass v1.2.0 // indirect
+	github.com/bep/logg v0.4.0 // indirect
+	github.com/caarlos0/env v3.5.0+incompatible // indirect
+	github.com/caarlos0/env/v11 v11.3.1 // indirect
+	github.com/cespare/xxhash/v2 v2.3.0 // indirect
+	github.com/clbanning/mxj/v2 v2.7.0 // indirect
+	github.com/cli/safeexec v1.0.1 // indirect
+	github.com/dgrijalva/jwt-go v3.2.0+incompatible // indirect
+	github.com/fatih/color v1.18.0 // indirect
+	github.com/gobwas/glob v0.2.3 // indirect
+	github.com/gohugoio/hashstructure v0.1.0 // indirect
+	github.com/gohugoio/hugo v0.140.2 // indirect
+	github.com/gohugoio/locales v0.14.0 // indirect
+	github.com/gohugoio/localescompressed v1.0.1 // indirect
```

Comment:
The presence of both v4 and v3 versions of `github.com/labstack/echo` suggests that you may have a conflict. Specify which version you intend to use directly in your code and consider removing the indirect one if not necessary.